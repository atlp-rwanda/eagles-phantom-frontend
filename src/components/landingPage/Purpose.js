import React, { Component }  from 'react';
import ShortAbt from './ShortAbt';
import '../../scss/homeContent.scss';
import { Button, Navbar, Nav, NavItem, NavDropdown, MenuItem } from 'react-bootstrap';

export default class Purpose extends Component {
    render(){
    return(
      <div className="purposeApp" >
          <h1 className="purposeApp__title">Welcome to Phantom App</h1>
          <Button className="travelBtn" as="input" type="submit" value="Travel with Us" />{' '}
          <div className="purposeApp__card">
              <ShortAbt
                  description="Track your bus with us wherever you’re in Kigali and explore the city without  going on the bus-stop lines"
              />
             <ShortAbt
                  description="Give us both your destination and location; we’ll help you know when the bus will pick you."
              />
          </div>
    </div>
  )
    }
  }