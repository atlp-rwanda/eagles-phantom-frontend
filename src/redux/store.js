import {createStore, applyMiddleware } from 'redux';
import { composeWithDevTools} from 'redux-devtools-extension';
import thunk from 'redux-thunk';
import promise from 'redux-promise-middleware';
import allReducers from './reducers/';

// const rootReducer = combineReducers({
//   count: counterReducer,
// });

// const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;

const middleware = applyMiddleware(thunk, promise)
const store = createStore(allReducers, {}, composeWithDevTools(middleware));

export default store;
