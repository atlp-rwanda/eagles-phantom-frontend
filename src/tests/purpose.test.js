import React from 'react';
import { Provider } from "react-redux";
import store from "../redux/store.js";
import { render } from '@testing-library/react';
import Purpose from '../components/landingPage/Purpose';

const purpose=()=>{
    return render(
       <Provider store={store}>
       <Purpose />
       </Provider>
    )
}
describe("Purpose",()=>{
    it("shoould render the component",()=>{
        const { asFragment }=purpose();
        expect(asFragment(<Purpose />)).toMatchSnapshot()
    });
})