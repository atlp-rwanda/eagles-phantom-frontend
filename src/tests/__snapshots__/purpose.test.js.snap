// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Purpose shoould render the component 1`] = `
<DocumentFragment>
  <div
    class="purposeApp"
  >
    <h1
      class="purposeApp__title"
    >
      Welcome to Phantom App
    </h1>
    <input
      class="travelBtn btn btn-primary"
      type="submit"
      value="Travel with Us"
    />
     
    <div
      class="purposeApp__card"
    >
      <div
        class="about"
      >
        <p>
          Track your bus with us wherever you’re in Kigali and explore the city without  going on the bus-stop lines
        </p>
      </div>
      <div
        class="about"
      >
        <p>
          Give us both your destination and location; we’ll help you know when the bus will pick you.
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
